pipeline/tasks/deploy-code.yml

```diff
@@ -0,0 +1,63 @@
+platform: linux
```

Comment:
Rename this to deploy-site (as it's a bit more specific to deploy-code)

Reply:
done